spring-cloud-alibaba-starters/spring-cloud-alibaba-sentinel-datasource/pom.xml

```diff
@@ -14,6 +14,10 @@
     <artifactId>spring-cloud-alibaba-sentinel-datasource</artifactId>
     <name>Spring Cloud Alibaba Sentinel DataSource</name>
 
+    <properties>
+        <opensergo-datasource.version>2.0.0-SNAPSHOT</opensergo-datasource.version>
```

Comment:
We could use `0.1.0-beta` version here.
Refer https://opensergo.io/zh-cn/docs/quick-start/java/quick-start-sentinel/

Reply:
Yeah, I will fix it